RUFAS/routines/field/crop/water_uptake.py

```diff
@@ -4,35 +4,44 @@
 from RUFAS.routines.field.crop.nitrogen_incorporation import NitrogenIncorporation
 from RUFAS.routines.field.soil.soil_data import SoilData
 
-"""
-This module is based upon the "Water Uptake By Plants" section of SWAT (5:2.2.1)
-"""
-
 # TODO: these methods do not currently account for whether or not the roots can reach a layer. See Nitrogen module.
 #   I'm not entirely sure if that should happen for this module, since the methods allow a crop to pull water up
 #   from deeper in the soil if the given layer does not have enough to meet the demands of that depth.
 
 
 class WaterUptake:
+    """
+    This module is responsible for all water uptake routines for a crop in a day.
+
+    Parameters
+    ----------
+    crop_data : Optional[CropData], optional
```

Comment:
```suggestion
    crop_data : CropData, optional
```
Consistent with first suggested change of this type.